test/Serializer/ArticleVoRNormalizerTest.php

```diff
@@ -428,11 +428,12 @@ function (ApiTestCase $test) {
                     promise_for(null), promise_for(new Copyright('CC-BY-4.0', 'Statement', 'Author et al')),
                     new ArraySequence([new PersonAuthor(new PersonDetails('Author', 'Author'))]), null,
                     promise_for(null), null, new ArraySequence([]), promise_for(null), new ArraySequence([
-                        new Section('Article 1 section title', 'article1section', [new Paragraph('Article 1 text')]),
+                        new Section('Article article1 section title', 'articlearticle1section', [new Paragraph('Article article1 text')]),
                     ]), new ArraySequence([
-                        new BookReference(ReferenceDate::fromString('2000'),
-                            [new PersonAuthor(new PersonDetails('preferred name', 'index name'))], false, 'book title',
-                            new Place(null, null, ['publisher'])),
+                        // minimum article-vor does not have reference, according to api-raml model
```

Comment:
I'll delete the comment then, there was an incomplete loop in `TestCase` which didn't check whether expected objects had too much stuff in them